spec/policies/user/user_policy_spec.rb

```diff
@@ -14,7 +14,7 @@
   describe 'how to handle concrete objects' do
     let(:object_to_check) { user }
 
-    it { is_expected.to permit_actions [:edit] }
-    it { is_expected.to forbid_actions [:new, :create, :update, :destroy, :index] }
+    it { is_expected.to permit_actions [:edit, :update] }
```

Comment:
Az ApplicationPolicy alapból tiltaná a show?-t. Szerintem érdemes lenne iderakni.

Reply:
Átfogalmazom.
- A sima users_controller alatt megszűnt a show endpont, mert business igény. :heavy_check_mark: 
- A users_policy-nak ezek alapján változnia kell. :red_circle: (Ha jól látom, akkor az admin/users_controller-ben továbbra is van egy show. Érdemes lehet akkor csak az adminokat beengedni erre az endpontra.)
- Ezek alapján meg fog változni a fenti teszt, hiszen tiltjuk a show-t a sima usereknek, hiszen nincs show endpontjuk.